docs/source/notebooks/iv_pymc.ipynb

```diff
@@ -0,0 +1,4160 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #6.    simple_ols_reg = sk_lin_reg().fit(X, y)
</pre><p>Not relevant for this PR but I think it would be nicer to test the bayesian estimates with statsmodels so that we get the usual stats like p-values and confidence intervals. WDYT?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-labs/CausalPy/pull/213/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
I have an ambition of specifically adding in statsmodels as a CausalPy backend, but I think you meant just to compare with a simple statsmodels output just for this example?

Reply:
Yeah, I wasn't sure of its status as a dependency and realised I could use sklearn to do 2SLS so I didn't have to risk adding any more dependencies.

Reply:
Let's create a discussion or issue to discuss this. As I mentioned initially, this is irrelevant to the PR 🙈 !